### PR TITLE
[skip ci]Convert SprintTag to pipeline

### PR DESCRIPTION
**Pull request by PengTian0**
Convert SprintTag to pipeline
Depends on https://github.com/RackHD/on-build-config/pull/103, https://github.com/RackHD/on-build-config/pull/104,  https://github.com/RackHD/on-build-config/pull/91,  https://github.com/RackHD/on-build-config/pull/90 

image

### CODE REVIEW

**Comment by PengTian0** [comment]
@panpan0000 @changev @anhou @yyscamper @keedya

File: jobs/SprintTag/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    checkout scm
+    if(TAG == "true"){
+        stage("Create Tag"){
+            load("jobs/SprintTag/create_tag.groovy")
+        }
+    }
+    def manifest_name=env.MANIFEST_FILE_URL.tokenize('/')[-1];
+    currentBuild.description = "<a href=${env.MANIFEST_FILE_URL}>${manifest_name}</a>";
+
+    def retry_times = Integer.valueOf(env.Test_Time);
```
Comment (panpan0000) [inline jobs/SprintTag/Jenkinsfile:12]: is ```Test_Time``` a job parameter ?

File: jobs/SprintTag/create_tag.groovy [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        }
+    
+        withCredentials([
+            usernameColonPassword(credentialsId: 'GITHUB_USER_PASSWORD_OF_JENKINSRHD', 
+                                  variable: 'JENKINSRHD_GITHUB_CREDS'),
+            usernamePassword(credentialsId: 'a94afe79-82f5-495a-877c-183567c51e0b', 
+                             passwordVariable: 'BINTRAY_API_KEY', 
+                             usernameVariable: 'BINTRAY_USERNAME')
+        ]){
+            if(JUMP_VERSION == "true"){
```
Comment (panpan0000) [inline jobs/SprintTag/create_tag.groovy:24]: Let me move this step to sprint-release-build pipeline.

Comment (panpan0000) [inline jobs/SprintTag/create_tag.groovy]: Let me move this step to sprint-release-build pipeline.

Comment (panpan0000) [inline jobs/SprintTag/create_tag.groovy:24]: Let me move this step to sprint-release-build pipeline.

File: jobs/SprintTag/Jenkinsfile (hunk 1 above)
Comment (PengTian0) [reply jobs/SprintTag/Jenkinsfile:12]: Yes

File: jobs/SprintTag/create_tag.groovy (hunk 2 above)
Comment (PengTian0) [reply jobs/SprintTag/create_tag.groovy:24]: OK!

File: jobs/SprintTag/create_tag.groovy [hunk 3]
```diff
@@ -0,0 +1,59 @@
+node{
+    withEnv([
+        "tag_name=${tag_name}",
+        "JUMP_VERSION=${JUMP_VERSION}",
```
Comment (panpan0000) [inline jobs/SprintTag/create_tag.groovy]: per @PengTian0 Tian
due to we have decided to "jump version" until the official release-tag added  ( jump was did when rc-tag added).
so the ${JUMP_VERSION} default being FALSE. which was set in Jenkins.

**Review by panpan0000 — APPROVED** [review]

File: jobs/SprintTag/create_tag.groovy (hunk 2 above)
Comment (PengTian0) [reply jobs/SprintTag/create_tag.groovy:24]: Done!

**Review by anhou — APPROVED** [review]

**Pull request merged**